### PR TITLE
Webpack sourcemaps

### LINKED ISSUE #94: Turn on sourcemaps
# Issue

Debugging is a pain right now:

image
# Fix

If sourcemaps are turned on, we'll get pre-bundled stack traces and source code to debug.  This is done with the top level [`devtool`](https://webpack.github.io/docs/configuration.html#devtool) key in webpack config:

``` js
{
  devtool: 'cheap-module-eval-source-map',
}
```

The suggestion here gives the fastest rebuilds (`++` on the chart) that also shows original source.  This is also the default for the next major version of Webpack.

### PR DESCRIPTION
**Pull request by FiberJW**
fixes #94

### CODE REVIEW

**Comment by FiberJW** [comment]
LGTM

File: webpack.config.build.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ module.exports = {
     UglifyJsPluginConfig,
     new webpack.optimize.DedupePlugin(),
   ],
+  devtool: 'cheap-module-eval-source-map',
```
Comment (eanplatter) [inline webpack.config.build.js:66]: Do we need sourcemaps in the prod build?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "enclave",
-  "version": "0.10.1",
+  "version": "0.10.2",
```
Comment (eanplatter) [inline package.json:3]: Don't worry about doing this, there's a release script we run off master which does this programmatically

**Comment by eanplatter** [comment]
A couple of comments to address, once those are fixed I'll approve!

File: webpack.config.build.js (hunk 1 above)
Comment (FiberJW) [inline webpack.config.build.js:66]: This is a good point. No we don't

**Pull request closed without merging**